src/app/core/components/spinner/spinner.component.html

```diff
@@ -1,3 +1,3 @@
-<div *ngIf="isLoading$ | async" class="spinner-container">
+<div *ngIf="isLoading$ | async" class="spinner">
```

Comment:
I believe, that spinner must be a dummy component which take `isLoading` boolean input and shows proper template. `isLoading$` must be held in `smart` component or `smart container`.

Reply:
Okay, will do